ts/Extensions/Annotations/Annotations.ts

```diff
@@ -77,6 +75,40 @@ const {
  * Declarations
  *
  * */
+
+declare module './AnnotationChart'{
+    interface AnnotationChart extends Chart {
+        annotations: Array<Annotation>;
+        controlPointsGroup: SVGElement;
+        options: Annotation.ChartOptions;
+        plotBoxClip: SVGElement;
+        addAnnotation(userOptions: AnnotationsOptions, redraw?: boolean): Annotation;
+        drawAnnotations(): void;
+        initAnnotation(userOptions: AnnotationsOptions): Annotation;
+        removeAnnotation(idOrAnnotation: (number|string|Annotation)): void;
+    }
+}
```

Comment:
`AnnotationChart` is only declared in this file, so this should be moved to `Annotation.Chart` namespace and the `AnnotationChart.d.ts` should be deleted to simplify things.